frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -180,8 +180,8 @@
   "Change Subscription update channel": "Change Subscription update channel",
   "Which channel is used to receive updates?": "Which channel is used to receive updates?",
   "Operator <1>{{name}}</1> will be removed from <3>{{namespace}}</3>. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.": "Operator <1>{{name}}</1> will be removed from <3>{{namespace}}</3>. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.",
-  "The console plugin provided by this operator will be disabled and removed.": "The console plugin provided by this operator will be disabled and removed.",
-  "The console plugin provided by this operator will be disabled and removed._plural": "The console plugins provided by this operator will be disabled and removed.",
+  "The console plugin provided by this operator will be disabled and removed._one": "The console plugin provided by this operator will be disabled and removed.",
+  "The console plugin provided by this operator will be disabled and removed._other": "The console plugin provided by this operator will be disabled and removed.s",
```

Comment:
`plugins`  parsed incorrectly for plurral

Reply:
> `plugins` parsed incorrectly for plural

I think this is a bug. The `count` property is unused - https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx#L212